habitat_baselines/common/construct_vector_env.py

```diff
@@ -40,10 +41,14 @@ def construct_envs(
 
     scene_splits: List[List[str]] = [[] for _ in range(num_environments)]
     if len(scenes) < num_environments:
-        logger.warn(
-            f"There are less scenes ({len(scenes)}) than environments ({num_environments}). "
-            "Each environment will use all the scenes instead of using a subset."
-        )
+        msg = f"There are less scenes ({len(scenes)}) than environments ({num_environments}). "
+        if is_eval:
+            raise ValueError(msg + "This is invalid for evaluation.")
```

Comment:
Why do we want to raise an error? Should we just reduce the number of environments accordingly if you really don't want the environments to have all scenes?

Reply:
Sure. I can just change the number.